### ISSUE TITLE
Tag confusion with Wiener thresholds

### ISSUE THREAD

**Issue by brettviren** [WireCell/wire-cell-toolkit#220]
The sigproc with Wiener filter produces "thresholds" which are RMS values.  These "threshods" are passed in a WCT frame in a "tagged trace summary".  The tagging involved has gone slightly off the rails.  I believe this is the situation, showing config params with typical values:

- `OmnibusSigProc` tags traces `wiener_tag="wiener"` and saves with no summary.
- `OmnibusSigProc` tags traces `wiener_threshold_tag="threshold"` with summary and using **same traces** as `wiener`.  This is the origin of the RMS values.
- `CellTreeSource` tags traces `wiener` (via map from ROOT branch name to tag name) with summary holding RMSes.
- `MaskSlice` reads traces **and** summary via single `wiener_tag`.

As is, `MaskSlice` is incompatible with `OmnibusSigProc` unless some game is played.  One of:

- Require `CellTreeSource` for its inclusion of "thresholds" into `wiener` tag.
- Use `Retagger` to ignore `wiener` and rename `threshold` to `wiener`.
- Configure `OmnibusSigProc` to "dump" the Wiener traces to some junk tag and use `wiener` for "threshold".
- Eradicate `threshold` tag everywhere and make `OmnibusSigProc` place the "threshold" summary in `wiener`.

I like the last option but worry about surprises.

**Comment by HaiwangYu** [comment]
Yes, I didn't notice this `OmnibusSigProc` behavior. I also like the last option. I think it may only affect uboone. And uboone is not using newer wct version. So maybe OK. Let me think a bit on this.

**Comment by brettviren** [comment]
After some discussion we will remove `threshold` tag and put its summary onto `wiener`.

**Comment by brettviren** [comment]
- [ ] `wclsFrameSaver` has an option `summary_tags:['threshold']` which is used to both read tagged trace summary from the `IFrame` and to name the `art::Event` product.  Simply changing this to `"wiener"` would still save the "threshold" summary but under the new `art::Event` location `"wiener"`.  Either `wclsFrameSaver` must be extended to take separate input and output names (ie, keeping `threshold` location in `art::Event`) or it is changed to `"wiener"` in `art::Event` and downstream code must adapt.
